src/job.rs

```diff
@@ -192,8 +192,7 @@ impl<'a, D> JMapExt<'a, D> for JMap<String, JValue>
     fn remove_datetime(&mut self, key: &str) -> Result<DateTime<Utc>, D::Error> {
         self.remove(key)
             .and_then(|v| v.as_f64())
-            .and_then(|f| NaiveDateTime::from_timestamp_opt(f as i64, ((f - f.floor()) * 1e9) as u32))
-            .map(|t| DateTime::from_utc(t, Utc))
+            .and_then(|f| DateTime::from_timestamp(f as i64, ((f - f.floor()) * 1e9) as u32))
```

Comment:
This resolves a deprecation in chrono. `from_utc` is also deprecated, but there's no need to use its alternative since [`DateTime::from_timestamp`](https://docs.rs/chrono/latest/chrono/struct.DateTime.html#method.from_timestamp) already returns a UTC-relative timestamp.